src/applications/static-pages/createCallToActionWidget.js

```diff
@@ -2,20 +2,18 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
 
-export default function createCallToActionWidget(store) {
+export default async function createCallToActionWidget(store) {
   const widgets = Array.from(document.querySelectorAll('.cta-widget'));
 
   if (widgets.length) {
-    widgets.forEach(async el => {
-      await import('../../platform/site-wide/cta-widget/sass/cta-widget.scss');
```

Comment:
Didn't quite get this style of importing the stylesheet to work, and I figure the work on the alerts in the Design System should properly style them anyway.